### PR TITLE
{numlib}[foss/2017a,intel/2017a] arpack-ng v3.4.0, arpack-ng v3.4.0

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)
arpack-ng 3.4.0 for foss/2017a and intel/2017a. No other changes.

### CODE REVIEW

File: easybuild/easyconfigs/a/arpack-ng/arpack-ng-3.4.0-foss-2017a.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+source_urls = ['https://github.com/opencollab/arpack-ng/archive/']
+sources = ['%(version)s.tar.gz']
+
+builddependencies = [('Autotools', '20150215')]
+
+preconfigopts = "sh bootstrap && "
+configopts = '--with-pic --with-blas="$LIBBLAS" --with-lapack="$LIBLAPACK"'
+
+sanity_check_paths = {
+    'files': ["lib/libarpack.a", "lib/libarpack.so"],
```
Comment (wpoely86) [inline easybuild/easyconfigs/a/arpack-ng/arpack-ng-3.4.0-foss-2017a.eb]: use `SHLIB_EXT`

File: easybuild/easyconfigs/a/arpack-ng/arpack-ng-3.4.0-intel-2017a.eb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+source_urls = ['https://github.com/opencollab/arpack-ng/archive/']
+sources = ['%(version)s.tar.gz']
+
+builddependencies = [('Autotools', '20150215')]
+
+preconfigopts = "sh bootstrap && "
+configopts = '--with-pic --with-blas="$LIBBLAS" --with-lapack="$LIBLAPACK"'
+
+sanity_check_paths = {
+    'files': ["lib/libarpack.a", "lib/libarpack.so"],
```
Comment (wpoely86) [inline easybuild/easyconfigs/a/arpack-ng/arpack-ng-3.4.0-intel-2017a.eb]: the same

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2428.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/cd895e51a9d2b77a265bd75c18fcb6bb for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 4 (2 easyconfigs in this PR)
node2003.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/2d9d5799b7cd90050eac62ca41449ce9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2124.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/017e130f8b5cebd5b3e7f0bd98ad80d9 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**